content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/image.html

```diff
@@ -26,6 +26,7 @@
      class="cmp-image${!wcmmode.disabled ? ' cq-dd-image' : ''}"
      itemscope itemtype="http://schema.org/ImageObject">
     <a data-sly-unwrap="${!image.link}"
+       data-sly-attribute.target="${image.linkTarget ? '_blank' : ''}"
```

Comment:
`data-sly-attribute` isn't necessary here, see [0]. I would remove that and place this after the `href`

[0] - https://github.com/adobe/aem-core-wcm-components/wiki/HTL-Style-Guide#avoid-the-element-attribute-and-text-block-statements

Reply:
Done